documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,294 @@
+
+.Validation for RHEL 6
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system, and therefore RHEL 6 is not supported in {ocp-name} Virtualization. A validation of RHEL 6 guest operating system was added. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+.Block migrations from {ocp-name} versions before 4.13
+
+The `Export API` in {ocp-name} Virtualization exposes VM manifests from {ocp-name} 4.13, so source {ocp} clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+.Source and storage endpoints are not selected automatically
+
+Source network and source storage are not selected automatically when creating an {ocp} to {ocp} plan. link:https://issues.redhat.com/browse/MTV-729[MTV729]
+
+.Restored vSphere Verify certificate functionality
+
+The vSphere Verify certificate option in the `add-provider` dialog has been restored. It was removed in the transition to {ocp} console. link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+.{ocp-name} Console native SDK fetch, post, and delete calls
+
+The `fetch`, `post`, and `delete` calls in the Forklift UI are now native to the {ocp-name} API. link:https://issues.redhat.com/browse/MTV-365[(MTV-365)]
+
+.{project-short} UI plugin uses only a dynamic plugin for the SDK REST API `fetch` methods
+
+The UI plugin uses a dynamic plugin for the SDK REST API when using the `fetch` methods. link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
+
+.Migration success status bar chart 
+
+A bar chart shows the number of migrations and their status - `starts`, `finished`, and `failed`. link:https://issues.redhat.com/browse/MTV-770[(MTV-770)]
+
+.Improved list of VMs in the detailed view of each provider
+
+The detailed view of each provider includes specific information relevant to the VMs of that provider. link:https://issues.redhat.com/browse/MTV-772[(MTV-772)]
+
+.Edit plans that failed to migrate VMs
+
+Plans that have failed to migrate any VMs can be edited. Some plans fail or are canceled because the network and storage mappings are wrong. These plans can be edited until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
+
+.ESXi provider host configuration
+
+ESXi, as a vSphere provider, does not need credentials in its setup. The credentials requirements field is removed from ESX settings. link:https://issues.redhat.com/browse/MTV-793[(MTV-793)]
+
+[id="ova-provider-26_{context}"]
+=== OVA Provider
+
+This is a collection of new features for the {project-short} OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+.Get firmware detection by virt-v2v
+
+A `virt-v2v` that converts a guest also produces an OFV with configuration based on the virtual disk, such as driver or firmware. `forklift-controller` processes the produced OFV's status of the conversion pod. link:https://issues.redhat.com/browse/MTV-759[(MTV-759)]
+
+.Delete PV when the provider is deleted
```

Comment:
```suggestion
.PV can is now automatically deleted when the provider is deleted
```
Correct me if I messed up the technical meaning :-D

Reply:
changed